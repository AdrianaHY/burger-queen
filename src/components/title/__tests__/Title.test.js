import React from 'react';
import { shallow } from 'enzyme';
import Title from '../Title';

describe('components | title | Title', () => {
	it('Renders the component', () => {
		const props = { title: 'Title'};

		const component = shallow( <Title {...props}/>)

		expect(component).toMatchSnapshot();
	});
});