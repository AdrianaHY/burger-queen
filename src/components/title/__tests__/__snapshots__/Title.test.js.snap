// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | title | Title Renders the component 1`] = `ShallowWrapper {}`;
